LearningMissionSimulation/PlaygroundGavril.cs

```diff
@@ -237,7 +262,39 @@ public void CreateClassrooms(int classroomCount)
 
         public void AssignInstructorsToClassrooms()
         {
-            throw new NotImplementedException();
+
+            string action = "Assign Instructors To Classrooms";
+            ReportHeader(actionName: action);
+
+            if (classroomList.Count == 0)
+            {
+                ReportError(missingResource: "Classroom", failedAction: action);
+            }
+            else
+            {
+                foreach (var classroom in classroomList)
+                {
+                    List<Instructor> instructorList;
+                    if (ModuleInstructorDictionary.TryGetValue(classroom.Module.Id, out instructorList))
+                    {
+                        if (instructorList.Count > 0)
+                        {
+                          var i =  classroom.Head = instructorList[AttributeGenerator.random.Next(0, instructorList.Count)];
+                        }
+                        else
+                        {
+                            ReportError(missingResource: "Instructor", failedAction: action);
+                        }
+                    }
+                    else
+                    {
+                        ReportError(missingResource: "Instructor", failedAction: action);
+                    }
+                   
+                }
+            }
+            ReportSummary(actionName: action, itemCount: classroomList.Count  );// Classrumneri qanak vor@ stanuma instructrner
```

Comment:
You did not fix it. I want to see here a number of classrooms for which you were  able to find a head (instructor).